src/components/buttons/FlipButton.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+import type { Node } from 'react';
+
+import styled from 'styled-components';
+import { Button } from 'lattice-ui-kit';
+
+const RotationWrapper = styled.div`
+  > button {
+    background-color: white;
+    border: none;
+    padding: 8px 12px;
+    transform: ${(props) => (props.flip ? 'rotate(180deg)' : 'rotate(0deg)')};
+    transition: transform 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+`;
+
+type Props = {
+  children :Node;
+  className :string;
+  flip :boolean;
+  onClick ?:() => void;
+  size ?:string;
+};
+
+const FlipButton = (props :Props) => {
```

Comment:
I think you might not need this component. There already exists a `Flip` component and this is how I use it:

```
<Flip flip={isOpenSearchOptions}>
  <IconButton aria-label="toggle search options" onClick={toggleSearchOptions}>
    <FontAwesomeIcon fixedWidth icon={faChevronDown} size="xs" />
  </IconButton>
</Flip>
```

You can see this in use in `OrgContainer`:

<img width="194" alt="Screen Shot 2021-06-02 at 3 35 17 PM" src="https://user-images.githubusercontent.com/440299/120560748-2362ac00-c3b8-11eb-8927-ceca5f473d1b.png">

Would this work for you?